### ISSUE TITLE
Cannot read property 'default' of undefined

### ISSUE THREAD

**Issue by zolcman** [mironov/react-redux-loading-bar#51]
> Warning: performUpdateIfNecessary: Unexpected batch number (current 2, pending 1)
> 
> connectAdvanced.js?4805:241 Uncaught TypeError: Cannot read property 'default' of undefined
> Function.mapStateToProps [as mapToProps] (loading_bar.js?3859:250)

Try add redux loading bar and have this error

My reducer

```
import { combineReducers } from 'redux-immutable';
import { routerReducer as router } from 'react-router-redux';

import { loadingBarReducer } from 'react-redux-loading-bar'
export default combineReducers({

  router,
  ProtectedReducer,
  loadingBar: loadingBarReducer,

});

How I add

import LoadingBar from 'react-redux-loading-bar'
import { ImmutableLoadingBar as LoadingBar } from 'react-redux-loading-bar' - not helps to
and in render I add <LoadingBar  />

**Comment by mironov** [comment]
@zolcman This should be fixed in v3.0.2.

Please use the bundled `immutableLoadingBarReducer` with an immutable store.

The commit for the reference: https://github.com/mironov/react-redux-loading-bar/commit/45ced448dde5379fe24e59fdc36c4d08bb1e94d8

**Comment by mironov** [comment]
@zolcman I just released v3.0.3 that doesn't require to use `immutableLoadingBarReducer `.
See the following commit as a reference implementation: https://github.com/mironov/react-redux-loading-bar/commit/c66900a1df2c9e91ec3db32856f349562a592369
